common/borders/borders-fr.html

```diff
@@ -0,0 +1,77 @@
+---
+{
+	"title": "Bordures",
+	"language": "fr",
+	"altLangPage": "borders-en.html",
+	"breadcrumbs":
+	[
+		{
+			"title": "Accueil GCWeb",
+			"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-fr.html"
+		}
+	],
+	"secondlevel": false,
+	"dateModified": "2023-06-20",
+	"share": "true"
+}
+---
+
+<h3>Classe CSS</h3>
+<dl class="dl-horizontal">
+	<dt><code>.brdr-left</code></dt>
+	<dd>Établir une bordure à gauche de l'élément</dd>
+	<dt><code>.brdr-tp</code></dt>
+	<dd>Établir une bordure en haut de l'élément</dd>
+	<dt><code>.brdr-rght</code></dt>
+	<dd>Établir une bordure à droite de l'élément</dd>
+	<dt><code>.brdr-bttm</code></dt>
+	<dd>Établir une bordure en bas de l'élément</dd>
+	<dt><code>.brdr-0</code></dt>
+	<dd>Supprimer les bordures</dd>
+	<dt><code>.brdr-rds</code></dt>
+	<dd>Supprimer le rayon de la bordure</dd>
+</dl>
+<div class="row">
+	<div class="col-md-6">
+		<div class="panel panel-default">
+			<div class="panel-body">
+				<h3 class="mrgn-tp-0">Apparence</h3>
+				<p class="brdr-lft">
+					Exemple 1
+				</p>
+				<p class="brdr-tp">
+					Exemple 2
+				</p>
+				<p class="brdr-rght">
+					Exemple 3
+				</p>
+				<p class="brdr-bttm">
+					Exemple 4
+				</p>
+				<p class="brdr-0">
+					Exemple 5
+				</p>
+				<p class="brdr-rds-0">
+					Exemple 6
+				</p>
+			</div>
+		</div>
+		<div class="panel panel-default">
```

Comment:
I think the intend was to have it side by side so this is missing

```suggestion
		</div>
	</div>
	<div class="col-md-6">
		<div class="panel panel-default">
```

Note: I am assuming that you are also removing the panel for the code sample, I kept it here to show how to configure the Grid